bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccBindingConstants.java

```diff
@@ -31,6 +31,8 @@ public class EvccBindingConstants {
     public static final ThingTypeUID THING_TYPE_DEVICE = new ThingTypeUID(BINDING_ID, "device");
 
     // List of all Channel Type UIDs
+    public static final ChannelTypeUID CHANNEL_TYPE_UID_BATTERY_CAPACITY = new ChannelTypeUID(BINDING_ID,
+            "batteryCapacity");
```

Comment:
Just curious, why not use the already defined constant here?
```suggestion
            CHANNEL_BATTERY_CAPACITY);
```

Reply:
Good idea!

Reply:
Done, see https://github.com/openhab/openhab-addons/pull/14245/commits/c9811852f2db1f6b45e94a315311628460170ec4.